drivers/spi/Kconfig

```diff
@@ -25,7 +25,7 @@ menuconfig SPI
 
 config SPI_QMSI
 	bool "QMSI driver for SPI controller"
-	depends on SPI && QMSI
+	depends on SPI && QMSI && SPI_LEGACY_API
```

Comment:
These Kconfig fixes should be in another patch

Reply:
what's platforms are exactly tested in the shippable builds for the enc28j60?